src/models/kernels/reservoir/Reservoir_Linear_Outlet.cpp

```diff
@@ -0,0 +1,22 @@
+#include "Reservoir_Linear_Outlet.hpp"
+
+/**
+ * @brief Default Constructor building an empty Reservoir Linear Outlet Object.
+ */
+Reservoir_Linear_Outlet::Reservoir_Linear_Outlet() : Reservoir_Outlet()
+{
+
+}
+
+/**
+ * @brief Parameterized Constructor that builds a Linear Reservoir Outlet object.
+ * @param a outlet velocity calculation coefficient
+ * @param activation_threshold_meters meters from the bottom of the reservoir to the bottom of the outlet
+ * @param max_velocity_meters_per_second max outlet velocity in meters per second
+ */
+Reservoir_Linear_Outlet::Reservoir_Linear_Outlet(double a, double activation_threshold_meters,
+                             double max_velocity_meters_per_second) :
+        Reservoir_Outlet(a, 1.0, activation_threshold_meters, max_velocity_meters_per_second)
```

Comment:
Reservoir linear outlet class inherits from outlet class and sets the exponent "b" to 1 at construction to create linearity in the outlet velocity function used from the outlet class. Should the linear class instead override the outlet velocity function with a modified equation that does not have the exponent "b" to create a linear equation?

Reply:
Might not be a bad idea just to show it explicitly.